web/packages/teleterm/src/ui/services/keyboardShortcuts/getKeyName.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Based on tabby https://github.com/Eugeny/tabby/blob/7a8108b20d9cbaab636c932ceaf4bacc710d6a40/tabby-core/src/services/hotkeys.util.ts
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Eugene Pankov
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ */
+
+const REGEX_LATIN_KEYNAME = /^[A-Za-z]$/;
+
+/**
+ * Returns a key name in a way that makes some keys independent of their physical placement on the US QWERTY layout.
+ *
+ * On the beginning, we check if the printed character is from the range A-Z (case-insensitive).
```

Comment:
```suggestion
 * First we check if the printed character is from the range A-Z (case-insensitive).
```

I wasn't sure if it's "at the beginning" or "in the beginning" but we can use "first" instead.